setup.py

```diff
@@ -75,6 +77,7 @@
     "kafka-python~=1.4",
     "sklearn-crfsuite~=0.3.6",
     "PyJWT~=1.7",
+    "setuptools>=41.0.1",
```

Comment:
can you please add a link to a docs page / issue / stackoverflow post for us to remember why this is in here?

also do we need 41.0.1 or does 41.0.0 work as well?

Reply:
hm, removed it, travis seems to be passing now. Do not remember why I added them 🙈